pysatNASA/instruments/methods/ace.py

```diff
@@ -43,8 +98,15 @@ def clean(self):
 
     """
 
+    # Get a list of coords for the data
+    if self.pandas_format:
+        coords = [self.data.index.name]
+    else:
+        coords = [key for key in self.data.coords]
```

Comment:
```suggestion
        coords = [key for key in self.data.coords.keys()]
```

Stylistically, we haven't been doing the implied key output.